app/components/Form/ApplicationForm.tsx

```diff
@@ -191,11 +176,28 @@ const ApplicationForm: React.FC<Props> = ({
   const isWithdrawn = status === 'withdrawn';
   const isDraft = status === 'draft';
 
+  const isSubmitEnabled = useMemo(() => {
+    if (isWithdrawn) return false;
```

Comment:
I don't mind skipping curly brackets for a single return statement but this function feels weird with them all removed.

Reply:
Would blank lines between the conditions help readability? 

```ts
  const isSubmitEnabled = useMemo(() => {
    if (isWithdrawn) return false;

    if (sectionName === 'review')
      return noErrors || formData.review?.acknowledgeIncomplete;

    if (sectionName === 'acknowledgements')
      return areAllAcknowledgementsChecked;
      
    if (sectionName === 'submission') return areAllSubmissionFieldsSet;

    return false;
  }, [
    sectionName,
    noErrors,
    areAllAcknowledgementsChecked,
    areAllSubmissionFieldsSet,
    isWithdrawn,
    formData,
  ]);
```

Reply:
It's fine as is just looks more like ruby than js :p